mathsnuggets/widgets/taylor.py

```diff
@@ -0,0 +1,20 @@
+import sympy
+
+from mathsnuggets.core import fields, form
+
+test = {"function": "x sin x"}
+
+
+class Taylor(form.Form):
+    """Taylor Expansion"""
+
+    template = """
+            Expand `expression``expand`
+    """
+
+    expression = fields.Expression("Expression")
+    x = fields.Expression("x", default="x")
+
+    @fields.computed("Taylor")
+    def taylor(self):
+        return sympy.series(self.expression)
```

Comment:
series takes additional parameters: the variable with respect to which you expand (x), the point around which we perform the expansion (so far, we have only used 0), and the order.

I would add fields for all these parameters: x0 (point around which we expand, default value 0), and order.

`sympy.series.series.series(expr, x=None, x0=0, n=6, dir='+')`

Then, I think it is sympy.series(self.expression, x=self.x, x0=self.x0, n=self.order)